src/main/java/com/ikubinfo/dto/PasswordDto.java

```diff
@@ -0,0 +1,30 @@
+package com.ikubinfo.dto;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Pattern;
+
+public class PasswordDto {
+	@NotEmpty
+	private String oldPassword;
+
+	@Pattern(regexp = "^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[@#$%^&+=])(?=\\S+$).{6,20}$", 
+	message = "Password should contain at least 6 characters, at least one lowercase letter at least one uppercase letter, at least a number and a special character, and no spaces allowed")
+	private String newPassword;
+
```

Comment:
ketu ne pergjithesi vendoset dhe confirm password, per ta pasur pak me safe, just in case e vendos gabim. thjesht per validim (sic eshte ne windows kur shkon te ndryshosh password)